cmd/krel/cmd/changelog.go

```diff
@@ -65,7 +71,7 @@ the golang based 'release-notes' tool:
    corresponding release-branch of kubernetes/kubernetes. The release branch
    will be pruned from all other CHANGELOG-*.md files which do not belong to
    this release branch.
-`,
+`, options.TokenKey),
```

Comment:
Is this always an env var? Specifically, `GITHUB_TOKEN`?
If so, I'd prefer we just call it what it is: `GitHubToken` or `TokenEnvVar`

Reply:
Alright, I renamed it to `GitHubToken` :+1: